cmd/api/README.md

```diff
@@ -11,6 +11,8 @@
 | DB_SSLMODE               | Database SSL mode                                                                                                                 | verify-full                                  |
 | DB_SSLROOTCERT           | Path to CA cert used to validate Database cert                                                                                    | /etc/tls/db/ca.crt                           |
 | DB_ENABLE_AUTO_MIGRATION | Auto-migrate the database on startup (create/update schemas). For further details, refer to <https://gorm.io/docs/migration.html> | true (default)                               |
+| PROFILING                | Enable profiling server                                                                                                           | false  (default)                             |
+| PROFILING_PORT           | Profiling Server Port                                                                                                             | 6060  (default)                              |
```

Comment:
fyi @sayan-biswas I just tried this and it looks like the `PROFILING_PORT` defaulting to `6060` does not take effect

```
{"level":"info","ts":1715275165.595052,"caller":"api/main.go:237","msg":"Profiling server listening on: "}
{"level":"info","ts":1715275165.5953324,"caller":"api/main.go:259","msg":"Prometheus server listening on: 9090"}
```

I tried 6060, 8008, and few others anyway, no luck.  Unfortunately, you can't run command like `netstat` in the container.


I'll try explicitly setting it to `6060` and see what happens

Reply:
it works OK when I explicitly set `PROFILING_PORT` @sayan-biswas